api/api/utils/throttle.py

```diff
@@ -116,14 +140,14 @@ def get_cache_key(self, request, view):
         if not (token_info and token_info.valid):
             return None
 
-        if token_info.rate_limit_model != self.applies_to_rate_limit_model:
+        if token_info.rate_limit_model not in self.applies_to_rate_limit_model:
             return None
 
         return self.cache_format % {"scope": self.scope, "ident": token_info.client_id}
 
 
 class OAuth2IdThumbnailRateThrottle(AbstractOAuth2IdRateThrottle):
-    applies_to_rate_limit_model = "standard"
+    applies_to_rate_limit_model = ["standard", "enhanced"]
```

Comment:
"enhanced" previously had no thumbnail rate applied to it. I don't think that's the correct behaviour, but I could be wrong. This change makes enhanced tokens have the same thumbnail rate limit as standard oauth. I think that's right because we have no separate thumbnail rate limit configured for enhanced tokens.

Exempt tokens, I assume, should be exempted from all rate limiting, so there's no change for that. That already worked.